complex/ft_config.c

```diff
@@ -776,6 +776,9 @@ void fts_cur_info(struct ft_series *series, struct ft_info *info)
 	info->ep_type = set->ep_type[series->cur_ep];
 	info->av_type = set->av_type[series->cur_av];
 	info->comp_type = set->comp_type[series->cur_comp];
+	if (info->caps & (FT_CAP_RMA | FT_CAP_ATOMIC) &&
+		(info->comp_type == FT_COMP_CNTR))
```

Comment:
Hmm... should this be ft_use_cntr_comp(), or is the check correct?

Reply:
That is before the commit that changed it to ft_use_cntr_comp(), the last commit changes this to use that function.

if (info->caps & (FT_CAP_RMA | FT_CAP_ATOMIC) &&
 +		(ft_use_comp_cntr(info->comp_type)))
 +		info->caps |= FI_RMA_EVENT;

Reply:
Thanks - I was reading the commits backwards.  :)